File: src/file.py
import json

def save_file(path="", date="", content=list):
    with open(path, "w") as file:
        json.dump(
            {
                "date": date,
                "content": content
            }, 
            file, 
            indent=4
        )

def get_file(path=""):
    with open(path, "r") as file:
        return json.load(file)

File: resources/js/file.js
const PATH = "/tasks.json"

const date = new Date()

async function write(content){
    const data = {
        date: `${date.getDate()}/${date.getMonth()+1}/${date.getFullYear()}`,
        content: content
    }
    await Neutralino.filesystem.writeFile(NL_PATH + PATH, JSON.stringify(data))
}

async function read(){
    let data = {
        date: `${date.getDate()}/${date.getMonth()+1}/${date.getFullYear()}`,
        content: []
    }
    try{
        data = JSON.parse(await Neutralino.filesystem.readFile(NL_PATH + PATH))
    }
    catch(err){}

    if(`${date.getDate()}/${date.getMonth()+1}/${date.getFullYear()}` != data.date){
        data.content.forEach(task=>task.checked = false)
    }
    return data
}